spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/tf/TensorflowLoaderSpec.scala

```diff
@@ -103,6 +104,19 @@ class TensorflowLoaderSpec extends TensorflowSpecHelper{
     System.setProperty("bigdl.enableNHWC", "false")
   }
 
+  "TensorFlow loader" should "convert tensor protobuf to bigdl tensor" in {
```

Comment:
This is just for debug. I'll remove this.

Reply:
I think it's fine for a test case